src/pattern-library/components/LibraryHome.js

```diff
@@ -5,35 +5,42 @@ export default function LibraryHome() {
   return (
     <Library.Page title="Pattern Library">
       <p>
-        This pattern library demonstrates foundations, patterns and components
-        that can be used in Hypothesis front-end applications.
+        This pattern library demonstrates UI toolkit components and how to use
```

Comment:
Content for this landing page will continue to become more meaningful through the component-update project.